src/Http/Http/src/Features/FormOptions.cs

```diff
@@ -16,83 +16,124 @@ public class FormOptions
         /// <summary>
         /// Default value for <see cref="MemoryBufferThreshold"/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 64KB.
```

Comment:
Could you remove the `<c></c>` tags? I don't think our other default docs have these. Typically they are only used for keywords like `<c>null</c>` or `<c>false</c>`.